### PR TITLE
537/1 Just making sure that people use localhost

### PR DESCRIPTION
**Pull request by karlcow**

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -40,4 +46,5 @@ def config_validator():
         print("Starting server in ~*TEST MODE*~")
         app.run()
     else:
+        print("[Use http://localhost:5000/]")
```
Comment (miketaylr) [inline run.py]: Did you mean to print `START_MESSAGE` here?

Comment (karlcow) [inline run.py]: rha… my bad. It was a remaining of a previous test.
You may erase the start message.

**Comment by karlcow** [comment]
Fixing my dirty laundry.

**Comment by miketaylr** [comment]
Yay, thanks!

**Pull request merged**